src/ZODB/DB.py

```diff
@@ -555,12 +559,15 @@ def f(con, detail=detail, rc=sys.getrefcount, conn_no=conn_no):
                 # sys.getrefcount(ob) returns.  But, in addition to that,
                 # the cache holds an extra reference on non-ghost objects,
                 # and we also want to pretend that doesn't exist.
+                # If we have no way to get a refcount, we return False to symbolize
+                # that. As opposed to None, this has the advantage of being usable
+                # as a number (0) in case clients depended on that.
```

Comment:
This is used in a diagnostic method, which is usually, but not always, printed. It seemed useful to me to be able to distinguish the "can't get a refcount" case from the "nobody but the cache has this item" case.